src/client/app/requests.js

```diff
@@ -11,13 +11,13 @@ export const Urls = {
     return url
   },
 
-  File: name => `/files/${name}`,
+  File: name => `/rtcstats-view/files/${name}`,
```

Comment:
the name of the api path path seems to be used throughout a bunch of places, could we move it to a constant?